docs/devtools.md

```diff
@@ -313,3 +313,21 @@ Examples commands and results:
 'schema1': {'PG': {'reporting_awscostentrylineitem_daily_summary': 'All data complete for table: reporting_awscostentrylineitem_daily_summary'}, 'TRINO': 'PG data complete so skipping trino query'}},
 'schema3': {'PG': {'reporting_awscostentrylineitem_daily_summary': 'All data complete for table: reporting_awscostentrylineitem_daily_summary'}, 'TRINO': 'PG data complete so skipping trino query'}}
 ```
+
+## Monitoring Celery ##
+
+[Flower] is a tool for monitoring Celery clusters. It provides detailed information about the status of workers and tasks.
+
+Flower is installed with the dev dependencies but it is not run by default. To start Flower, run
+
+```
+celery -A koku flower
```

Comment:
side note: it should be run inside koku/koku (or wherever the celery.py file is).